src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/OpenDocumentGeneratorTest.cs

```diff
@@ -166,8 +164,7 @@ await projectManager.UpdateAsync(updater =>
             updater.DocumentOpened(_hostProject1.Key, _documents[0].FilePath, SourceText.From(string.Empty));
 
             // Act
-            updater.ProjectWorkspaceStateChanged(_hostProject1.Key,
-                ProjectWorkspaceState.Create(LanguageVersion.CSharp8));
+            updater.ProjectWorkspaceStateChanged(_hostProject1.Key, ProjectWorkspaceState.Default);
```

Comment:
Same question as above.

Reply:
You raise a good point, and I think this test was already broken, otherwise this should actually be failing 🤦‍♂️